test/e2e/e2e.go

```diff
@@ -521,7 +552,7 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 		podChan, errChan := make(chan *v1.Pod), make(chan error)
 		go func() {
 			defer ginkgo.GinkgoRecover()
-			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "www.google.com.", 443, 30, podChan, errChan)
+			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "www.google.com.", 443, 10, 30, podChan, errChan)
```

Comment:
why did you change `timeout` to 10 here? everywhere else it's 30

Reply:
i think 10 is number of tries, but he did change (which is the same as before), but...
https://github.com/ovn-org/ovn-kubernetes/pull/2370/commits/30c882a7731b94ed84897f520b4d958fdf5c4753#diff-65b73a6a62a4e5b8903b9f17f0e03a42365b127e7d86f006f9bec363b4be04b8R489
to 30 tries